lib/adapter.js

```diff
@@ -26,7 +26,7 @@ var metrics = {
 
 var internal = {
   on: internal_on, // XXX(sam) where is this used?
-  supports: {watchdog: false} // XXX(sam) probably not needed
+  supports: {watchdog: process.platform === 'linux'}
```

Comment:
That makes sense, but currently in the adapter appmetrics doesn't exist until start() is called and the strong-supervisor tests need to check if watchdog is supported before that point (and presumably strong-arc does too via the capablities api). Do need to fix for ppc linux though, which will be a false positive at the moment.